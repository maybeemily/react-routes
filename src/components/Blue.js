import React from 'react';

export default function Blue() {
  const style = {
    backgroundColor: 'blue',
    width: '50vw',
    height: '50vw',
    fontSize: '4em',
    color: 'black'
  };

  return (
    <div style={style}>B L U E</div>
  );
}
