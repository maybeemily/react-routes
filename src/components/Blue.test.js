import React from 'react';
import { shallow } from 'enzyme';
import Blue from './Blue';

describe('Header component', () => {
  it('renders the header', () => {
    const wrapper = shallow(<Blue/>);
    expect(wrapper).toMatchSnapshot();
  });
});
