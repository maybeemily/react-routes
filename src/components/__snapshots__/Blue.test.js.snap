// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blue component renders blue div 1`] = `
<div
  style={
    Object {
      "backgroundColor": "blue",
      "color": "black",
      "fontSize": "4em",
      "height": "50vw",
      "letterSpacing": "1em",
      "width": "50vw",
    }
  }
>
  BLUE
</div>
`;
